### PR TITLE
Tidy up gemspec.

### PR DESCRIPTION
**Pull request by ioquatix**

### CODE REVIEW

**Comment by ioquatix** [comment]
@asppsa you asked about dropping 2.2 support and I think we will do that in the next release.

**Comment by ioquatix** [comment]
@jeremyevans if you are happy with this can you merge before doing the 2.2 release?

**Comment by ioquatix** [comment]
Also I think we should shut down the mailing list. People can use github issues.

File: rack.gemspec [hunk 1]
```diff
@@ -5,37 +5,37 @@ require_relative 'lib/rack/version'
...
+  s.description = <<~EOF
+    Rack provides a minimal, modular and adaptable interface for developing
+    web applications in Ruby. By wrapping HTTP requests and responses in
+    the simplest way possible, it unifies and distills the API for web
+    servers, web frameworks, and software in between (the so-called
+    middleware) into a single method call.
+  EOF
+
+  s.files = Dir['{bin/*,contrib/*,example/*,lib/**/*}'] +
+    %w(MIT-LICENSE rack.gemspec Rakefile README.rdoc SPEC)
```
Comment (jeremyevans) [inline rack.gemspec]: This should be `SPEC.rdoc` now. I missed this when I changed the name.

**Comment by jeremyevans** [comment]
Other than the SPEC -> SPEC.rdoc change, this looks fine.  I prefer Google Groups to GitHub Issues for non-bug reports, but I don't feel strongly about it.

**Comment by jeremyevans** [comment]
I fixed the SPEC filename after merging this.

**Comment by ioquatix** [comment]
Thanks. I was 💤.

**Pull request merged**